docs/apps/lnw/es2k/es2k-lnw-overlay-vms.md

```diff
@@ -31,21 +31,33 @@ for more details on this feature.
 
 Prerequisites:
 
-- Download `hw-p4-programs` TAR file specific to the build and extract it to get `fxp-net_linux-networking-v2` p4 artifacts. Go through `Limitations` specified in `README` and bringup the setup accordingly.
-- Follow steps mentioned in [Deploying P4 Programs for E2100](/guides/es2k/deploying-p4-programs) for bringing up IPU with a custom P4 package.
+- For Linux Networking V2
+  - Download `hw-p4-programs` TAR file specific to the build and extract it to get `fxp-net_linux-networking-v2` P4 artifacts. Go through `Limitations` specified in `README` and bring up the config accordingly.
```

Comment:
```suggestion
  - Download `hw-p4-programs` TAR file compliant with the CI build image and extract it to get `fxp-net_linux-networking-v2` P4 artifacts. Go through `Limitations` specified in `README` and bring up the config accordingly.
```

Couple of points:
* "specific to the build" is not clear, so posting a suggestion here for clarity
* "Go through Limitations ... and bring up setup" seems a bit odd way of phrasing for bringup. Perhaps a better way to say could be to "Check readme for bringup guide, and be sure to check limitation for known issues"?